_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -889,16 +889,197 @@ Terraform::
   Comprehensive Guide to Terraform], https://training.gruntwork.io/p/terraform[A Crash Course on Terraform], and
   link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to use the Gruntwork Infrastructure as Code Library].
 
+Terragrunt::
+  This guide uses https://terragrunt.gruntwork.io/[Terragrunt] to configure the infrastructure as code. To get familiar
+  with Terragrunt, explore the https://terragrunt.gruntwork.io/docs/#features[features], read the https://terragrunt.gruntwork.io/docs/getting-started/quick-start/[guides],
+  or dive into the https://terragrunt.gruntwork.io/docs/[documentation].
+
+Code repository::
+  You will need to initialize an `infrastructure-live` repository to contain all of the Terragrunt configuration code for your
+  infrastructure. You may use the https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/v0.41.4/examples/for-production/infrastructure-live[`for-production` example code] to start with.
+
 Keybase (optional)::
   As part of this guide, you will create IAM users, including, optionally, credentials for those IAM users. If you
   choose to create credentials, those credentials will be encrypted with a PGP key. You could provide the PGP keys
   manually, but a more manageable option may be to have your team members to sign up for https://keybase.io[Keybase],
   create PGP keys for themselves, and then you can provide their Keybase usernames, and the PGP keys will be retrieved
   automatically.
 
+
+=== Prepare your `infrastructure-live` repository
+
+[NOTE]
+.Terragrunt not required
+====
+This guide uses https://github.com/gruntwork-io/terragrunt[Terragrunt] and its associated file and folder
+structure to deploy Terraform modules. Please note that *Terragrunt is NOT required for using Terraform modules from
+the Gruntwork Infrastructure as Code Library.* Check out link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to use the Gruntwork Infrastructure as Code Library]
+for instructions on alternative options, such as
+link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library#deploy_using_plain_terraform[deploying with plain Terraform].
+====
+
+Now we're going to make some HCL files that store variables to be used across your modules. _You won't be able to fill
+everything out just yet._ Your AWS account IDs will be generated after applying the account-baseline-root to the root
+account. At that point you can update these files. Create them now to have them ready to use.
+
+----
+infrastructure-live
+  └ terragrunt.hcl
+  └ landingzone
+    └ account-baseline-root
+    └ account-baseline-app
+      └ main.tf
+      └ outputs.tf
+      └ variables.tf
+----
```

Comment:
Hm, I'm not sure this is applicable anymore given that we don't really use the `.tf` files anywhere in this guide as far as I could see. Is this intended?

Reply:
Good catch! I wonder why tf files are in here. Will investigate/remove.

Reply:
Oh, I know what I did. I meant to update this folder structure tree!

Reply:
Updated in 1c164411..698610fe.